phpunit-7.2-all.txt

```diff
@@ -0,0 +1,240 @@
+PHPUnit 8.5.22 by Sebastian Bergmann and contributors.
```

Comment:
What is this file?

Reply:
This is the output from a run of phpunit without filtering any of the tests out, on PHP 7.2.